app/main.py

```diff
@@ -1 +1,17 @@
-# write your code here
+import os
+
+
+def move_file(command: str) -> None:
+    parts = command.split()
+    source_path = parts[1]
+    destination_path = parts[2]
+    if not os.path.isfile(source_path):
+        raise FileNotFoundError(f"Source file {source_path} not found")
+    if destination_path.endswith("/"):
```

Comment:
Use os.path.split

Reply:
Also code is not works when dir in path but not ended by /

Reply:
"Also code is not works when dir in path but not ended by /"  Please explain this in more detail

Reply:
For example when I use the following command
`mv file.txt folder`

I expect the file to be moved to `folder` directory
But the code won't work because you assume that directory ends exactly with `/` symbol but it is not always true